### ISSUE TITLE
option.sigFile not work

### ISSUE THREAD

**Issue by issaTan** [stnvh/gulp-tinypng-compress#23]
Thanks for your work.
But here's a problem for me, my gulpfile is like this:
```
gulp.task('mobile-web-img', function () {
  return gulp.src('web/bundles/custommobileweb/img/**/*.{png, jpg, jpeg}')
    .pipe(tinypng({
      key: 'API_KEY',
      sigFile: 'web/bundles/custommobileweb/img/.tinypng-sigs',
      log: true,
      summarize: true
    })).on('error', function (err) {
      console.error(err.message);
    })
    .pipe(gulp.dest('web/bundles/custommobileweb/dist/img'))
});
```
but there's no `.tinypng-sigs` is created.
I have 200+ images in this folder, the plugin upload all images every time I  run the task.
And I tried set `sigFile: './.tinypng-sigs'` and created `.tinypng-sigs'` file myself, but it still not work.
What can I do? I've made something wrong?

And one more question, when it there's one image(e.g. name: a.png) `on error`, will the images before this one that had  compressed write into the `.tinypng-sigs` file?

**Comment by issaTan** [comment]
I think I found the reason, it was failed when compress some images, the task is not completed, so the plugin will not created the `. tinypng-sigs` file.
I think here's a bug. Cause the plugin compress 190+ images before it failed, but the next time it will still upload which had compressed, just because there's one image failed.

**Comment by stnvh** [comment]
Fixed in [1.2.2](https://github.com/stnvh/gulp-tinypng-compress/releases/tag/1.2.2) edit: (probably)

**Comment by stnvh** [comment]
Opening as this is a seperate issue

**Comment by stnvh** [comment]
Fixed in [1.2.3](https://github.com/stnvh/gulp-tinypng-compress/releases/tag/1.2.3)
